riff-raff.yaml

```diff
@@ -0,0 +1,20 @@
+stacks: [ support ]
+regions: [ eu-west-1 ]
+
+deployments:
+    dotcom-components-cloudformation:
+        type: cloud-formation
+        app: dotcom-components
+        parameters:
+            cloudFormationStackName: dotcom-components
+            templatePath: cloudformation/cfn.yaml
+            cloudFormationStackByTags: false
+            amiTags:
+                Recipe: support-dotcom-components-bionic
+                AmigoStage: PROD
+
+    dotcom-components:
+        type: autoscaling
+        dependencies: [ dotcom-components-cloudformation ]
+        parameters:
+            bucket: membership-dist
```

Comment:
@rupertbates I noticed here they are reusing the membership-dist bucket rather than adding a new one as you are planning for support zuora extractor job